cli/jobs/pipelines-with-components/basics/4b_datastore_datapath_uri/pipeline.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         type: uri_folder
         # format for data store uri: azureml://datastores/<datastore_name>/paths/<path>/<on>/<datastore>
         path: azureml://datastores/workspaceblobstore/paths/azureml
-        mode: rw_mount      
+        mode: ro_mount      
```

Comment:
@wangchao1230 this change will bring runtime error:
Failed one: [4b_datastore_datapath_uri - Microsoft Azure Machine Learning Studio](https://ml.azure.com/experiments/id/d994f1df-bb30-48df-851d-0366452353f9/runs/witty_wheel_460qn4shwk?wsid=/subscriptions/96aede12-2f73-41cb-b983-6d11a904839b/resourcegroups/cli-examples/workspaces/main-master&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#)
Completed one: 4b_datastore_datapath_uri - Microsoft Azure Machine Learning Studio